pkg/controllers/deprovisioning/events.go

```diff
@@ -12,7 +12,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package events
+package deprovisioning
```

Comment:
This is no longer idiomatic. The convention has been to create a separate `events` package in the corresponding controller directory so you can say

```go
deprovisioningevent.TerminatingNode(&v1.Node{})
```